src/msw/utils.cpp

```diff
@@ -1177,21 +1179,27 @@ wxString wxGetOsDescription()
                     break;
 
                 case 10:
-                    if (info.dwBuildNumber >= FIRST_WINDOWS11_BUILD)
+                    if ( wxIsWindowsServer() == 1 )
                     {
-                        str = wxIsWindowsServer() == 1
-                            ? "Windows Server 2022"
-                            : "Windows 11";
-
+                        switch ( info.dwBuildNumber )
+                        {
+                            case WINDOWS_SERVER2016_BUILD:
+                                str = "Windows Server 2016";
+                                break;
+                            case WINDOWS_SERVER2019_BUILD:
+                                str = "Windows Server 2019";
+                                break;
+                            case WINDOWS_SERVER2022_BUILD:
+                                str = "Windows Server 2022";
+                                break;
+                        }
```

Comment:
Should we do something like "Windows Server build %d" for the other, unknown, builds? I understand that this is not supposed to happen, but what if they release such build in the future?